dvc/main.py

```diff
@@ -84,7 +84,7 @@ def main(argv=None):  # noqa: C901
         ret = 255
 
     try:
-        if ret != 0:
+        if ret != 0 and ret != 1:
```

Comment:
I think this is no longer needed after the rebase
```suggestion
        if ret != 0:
```

Reply:
@efiop 
Still got this warning after I deleted `ret != 1`, is this OK?
![image](https://user-images.githubusercontent.com/6745454/88991992-ba805280-d314-11ea-8826-a656ef2fd1b7.png)

Reply:
@karajan1001, or just do something like this for now:
```python
if ret == 1 and args and args.cmd == "check-ignore":
    pass
```

Not ideal, but should work for now.